src/ui/public/index_patterns/_calculate_indices.js

```diff
@@ -2,6 +2,12 @@ define(function (require) {
   const _ = require('lodash');
   const moment = require('moment');
 
+  // gets parsed value if given arg is a moment object
+  function timeValue(val) {
+    const hasValueOfFn = _.isFunction(_.get(val, 'valueOf'));
```

Comment:
At the same time, any object can overwrite valueOf with whatever thing they want, so it's existence doesn't necessarily mean it's invokable.